app/models/sciencewire_source_record.rb

```diff
@@ -105,13 +105,13 @@ def self.get_sciencewire_source_record_from_sciencewire_by_sw_id(sciencewire_id)
   def self.get_and_store_sw_source_record_for_sw_id(sciencewire_id)
     sw_record_doc = ScienceWireClient.new.get_sw_xml_source_for_sw_id(sciencewire_id)
     pmid = extract_pmid(sw_record_doc)
-
     where(sciencewire_id: sciencewire_id).first_or_create(
       source_data: sw_record_doc.to_xml,
       is_active: true,
       pmid: pmid,
       source_fingerprint: Digest::SHA2.hexdigest(sw_record_doc))
   end
+  private_class_method :get_and_store_sw_source_record_for_sw_id
```

Comment:
On master:
```
$ git grep 'get_and_store_sw_source_record_for_sw_id'
app/models/sciencewire_source_record.rb:    get_and_store_sw_source_record_for_sw_id(sciencewire_id)
app/models/sciencewire_source_record.rb:  def self.get_and_store_sw_source_record_for_sw_id(sciencewire_id)
lib/tasks/cap_cutover.rake:      sciencewire_source_record = SciencewireSourceRecord.get_and_store_sw_source_record_for_sw_id(sciencewire_id)
```
This PR proposes to remove the public method call from `lib/tasks/cap_cutover.rake`.  Within the SW-src model, this does look like a private method, as there are some other methods that appear to be a public API.